src/pybind/mgr/dashboard/services/access_control.py

```diff
@@ -290,6 +290,7 @@ def __init__(self, username, password, name=None, email=None, roles=None,
         self.password = password
         self.name = name
         self.email = email
+        self.attempt = 0
```

Comment:
What about naming the variable something like that?
```suggestion
        self.invalid_login_attempts = 0
```
```suggestion
        self.invalid_auth_attempts = 0
```
IMO the current name `attempt` does not reflect the real intention what this variable is used for.

Reply:
Done.